cocos/core/renderer/scene/submodel.ts

```diff
@@ -399,7 +399,7 @@ export class SubModel {
         // update draw info
         const drawInfo = this._subMesh.drawInfo;
         if (this._inputAssembler && drawInfo) {
-            Object.assign(this._inputAssembler.drawInfo, drawInfo);
+            this._inputAssembler.drawInfo = drawInfo;
```

Comment:
I am not sure if `assign` and `=` have the same effect. I think there is some reason to use `assign` here. As SubModel is an important class, so it is better to do special logic for webgpu.